ui/apps/platform/eslint-plugins/pluginGeneric.js

```diff
@@ -0,0 +1,139 @@
+            },
+            schema: [],
+        },
+        create(context) {
+            return {
+                JSXElement(node) {
+                    if (node.openingElement?.name?.name === 'a') {
+                        const ancestors = context.sourceCode.getAncestors(node);
+                        if (
+                            ancestors.length >= 1 &&
+                            ancestors[ancestors.length - 1].openingElement?.name?.name !==
+                                'ExternalLink'
+                        ) {
+                            context.report({
+                                node,
+                                message:
+                                    'Require ExternalLink and a elements for consistent presentation',
+                            });
+                        }
+                    }
+                },
+            };
+        },
+    },
+    'Link-target-rel': {
+        // Require props for consistent behavior and security of internal links that open in a new tab.
+        meta: {
+            type: 'problem',
+            docs: {
+                description:
+                    'Require if Link element has target="_blank" that it has rel="noopener noreferrer" props',
+            },
+            schema: [],
+        },
+        create(context) {
+            return {
+                JSXOpeningElement(node) {
+                    if (node.name?.name === 'Link') {
+                        if (
+                            node.attributes.some(
+                                (nodeAttribute) =>
+                                    nodeAttribute.name?.name === 'target' &&
+                                    nodeAttribute.value?.value === '_blank'
+                            )
+                        ) {
+                            if (
+                                !node.attributes.some(
+                                    (nodeAttribute) =>
+                                        nodeAttribute.name?.name === 'rel' &&
+                                        nodeAttribute.value?.value === 'noopener noreferrer'
+                                )
+                            ) {
+                                context.report({
+                                    node,
+                                    message:
+                                        'Require if Link element has target="_blank" that it has rel="noopener noreferrer" props',
+                                });
+                            }
+                        }
+                    }
+                },
+            };
+        },
+    },
+    'a-target-rel': {
+        // Require props for consistent behavior and security of external links.
+        meta: {
+            type: 'problem',
+            docs: {
+                description:
+                    'Require that a element has target="_blank" rel="noopener noreferrer" props',
```

Comment:
nit: The text could be clearer here too. The "a" can be misinterpreted. 

```Require that <a> elements with target="_blank" include rel="noopener noreferrer"```

Reply:
Thanks yet again!